app/src/main/java/io/github/droidkaigi/confsched2017/model/Speaker.java

```diff
@@ -34,4 +39,20 @@
     @Nullable
     @SerializedName("github_name")
     public String githubName;
+
+    @Nullable
+    public String getAdjustedImageUrl() {
+        if (imageUrl == null) {
+            return null;
+        }
+
+        if (imageUrl.startsWith("http://") || imageUrl.startsWith("https://")) {
```

Comment:
In my opinion, I want to keep models independent as far as possible.
However in this case, we can use `URLUtil` since it's a simple utility class.
It's a matter of taste.